content/app-dev/workflow/_index.en.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Developer Workflows"
+date: 2018-12-29T11:02:05+06:00
+weight: 8
+draft: false
+---
+
+As a wasmCloud developer, there are a number of common day-to-day workflows that you will experience. Which workflows you run through and how often you do so depends entirely on what you're building. 
+
+The following is a list of developer workflows _sorted_ from **_most to least common_**.
+
+### Building Actors
+The most common thing application developers will do is build actors. Actors encompass pure business logic, and only communicate with non-functional requirements through capability providers and abstract [interfaces](/interfaces).
+
+Once you've established a dependency on a library that exposes the interface abstraction you're looking for, you can start your iteration loop.
+
+The developer's _iteration loop_ for building an actor looks something like this:
+1. Make code changes
+1. Compile and sign the WebAssembly module, creating a `_s.wasm` file (using `make` in an actor project)
+1. Deploy the module to public or local OCI registry (`make push`)
+1. Test the module in a host
+    1. Manually via uploading in wasmCloud web dashboard or with (`make start`)
+    1. Script `wash call` commands to invoke the actor
+    1. Leverage the `wasmcloud:testing` interface and the [test provider](https://github.com/wasmCloud/wasmcloud-test)
```

Comment:
We can use `make update` like the actor example to live update right?

Also, these should be bullet points as you can perform any of those actions, they aren't sequential